test/stream/api/source/version-detail-404/003-version-detail/expected-response.json

```diff
@@ -0,0 +1,5 @@
+{
+  "statusCode": 404,
+  "error": "stream.api.sourceVersionNotFound",
+  "message": "Source version \"my-source/0000000025\" not found in the branch."
```

Comment:
There should be one more step in this test - try accessing version detail before creating the source. The message should be different in that case.

Reply:
47f6964